README.md

```diff
@@ -34,3 +34,47 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI README](https://github.com/angular/angular-cli/blob/master/README.md).
 
 This project also has the backend AWS lambda code. Auto build is enabled. 
+
+## AWS Part
+
+## Creation of Dynamodb table.
+
+To log the id and update it with changes in form of adding items to that particular id, a table was created in data base in Dynamodb namely shared-list with primary key ‘id’.
+
+## Lambda functions.
+
+ Created three lambda functions from the code base namely:-
+ getshoppinglist, updateshoppinglist, CORS_function from the code base provided through github.
+ 
+ Getshoppinglist for generating an id and for calling with that id while updateshoppinglist for update the list with that given id and CORS_function to handle HTTP requests(get,put,options) and to allow CORS( 
+ Cross Origin Resource Sharing). 
+ Gave necessary IAM permissions in the form of role to the lambda functions so as toIntegrate it with the dynamodb table.
+ Testing the lambda function with appropriate payload (payload not for CORS_function)was done successfully.    
+
+## API gateway Integration
+
+ API Gateway enables to connect APIs to various backend services, such as AWS Lambda functions, Amazon EC2 instances, or other HTTP endpoints. For connecting lambda functions to APIs we have created an API 'sharedlist'.
+
+created the required routes for the api according with the lambda functions so here it is GET ( to get an id), GET {id} ( to get the id by calling a particular id), PUT( to update the shopping list associated with a particular id) and OPTIONS ( To describe the communication options for the target resource). 
+
+So Get and GET {id} was integrated with getshoppinglist lambda function and PUT on to updateshoppinglist function while OPTIONS on to CORS_function through
+‘Manage integration’.
+
```

Comment:
Add some screenshots here

Reply:
Unable to pin screenshot inside the file.